apps/tasks/lib/components/patient_bottom_sheet.dart

```diff
@@ -0,0 +1,175 @@
+                            onPressed: () {
+                              Navigator.of(context).pop();
+                            },
+                          ),
+                          Flexible(
+                            child: Padding(
+                              padding: const EdgeInsets.only(top: paddingOffset),
+                              child: Text(
+                                widget.task.patient.name,
+                                style: const TextStyle(
+                                  overflow: TextOverflow.ellipsis,
+                                  fontSize: fontSizeBig,
+                                ),
+                              ),
+                            ),
+                          ),
+                          IconButton(
+                            icon: const Icon(Icons.edit_outlined),
+                            onPressed: () {},
+                          ),
+                        ],
+                      ),
+                      SizedBox(
+                        width: width * 0.45,
+                        child: const Center(
+                          child: ExpansionTile(
+                            title: Text(
+                              overflow: TextOverflow.ellipsis,
+                              "Room 2 - Bed 1", // TODO: replace with real data
+                              style: TextStyle(color: Colors.grey),
+                            ),
+                            children: [
+                            ],
+                          ),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+                TextFormField(
+                  initialValue: widget.task.notes,
+                  decoration: InputDecoration(
+                    hintText: context.localization!.notes,
+                  ),
+                  maxLines: 6,
+                  keyboardType: TextInputType.multiline,
+                ),
+                Padding(
+                  padding: const EdgeInsets.symmetric(vertical: paddingMedium, horizontal: paddingOffset ),
+                  child:  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Text(context.localization!.tasks, style: const TextStyle(fontSize: fontSizeBig, fontWeight: FontWeight.bold),),
+                      Container(
+                        height: 40,
+                        width: 40,
+                        decoration: const BoxDecoration(
+                          shape: BoxShape.circle,
+                          color: primaryColor
+                        ),
+                        child: IconButton(
+                          onPressed: () {},
+                          icon: const Icon(size: 25 , Icons.add, color: Colors.white,),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
```

Comment:
Make the icon smaller maybe iconSizeSmall for the outer ring and iconSizeTiny for the add